modules/@apostrophecms/ui/ui/apos/lib/vue.js

```diff
@@ -1,14 +1,21 @@
-import Vue from 'vue';
+import { createApp } from 'vue';
 import ClickOutsideElement from './click-outside-element';
 import LocalizedVTooltip from './localized-v-tooltip';
 import tooltipOptions from './tooltip-options';
 import VueAposI18Next from './i18next';
 
-Vue.use(LocalizedVTooltip, tooltipOptions);
-Vue.use(ClickOutsideElement);
-Vue.use(VueAposI18Next, {
-  // Module aliases are not available yet when this code executes
-  i18n: apos.modules['@apostrophecms/i18n']
-});
+export default (appConfig) => {
+  const app = createApp(appConfig);
 
-export default Vue;
+  app.use(LocalizedVTooltip, tooltipOptions);
+  app.use(ClickOutsideElement);
+  app.use(VueAposI18Next, {
+    // Module aliases are not available yet when this code executes
+    i18n: apos.modules['@apostrophecms/i18n']
+  });
+
+  window.apos.registerVueComponents(app);
+  window.apos.registerIconComponents(app);
```

Comment:
These methods  are generated in the import file, the easiest way to be able to register all components for all apps.